examples/common/pigweed/rpc_services/Attributes.h

```diff
@@ -33,14 +33,34 @@
 #include <app/data-model-provider/Provider.h>
 #include <app/util/attribute-storage.h>
 #include <app/util/attribute-table.h>
+#include <data-model-providers/codegen/CodegenDataModelProvider.h>
 #include <lib/core/TLV.h>
 #include <lib/core/TLVTags.h>
 #include <lib/core/TLVTypes.h>
+#include <pigweed/rpc_services/AttributeAccessor.h>
+#include <pigweed/rpc_services/AttributeAccessorRegistry.h>
 #include <platform/PlatformManager.h>
+#include <set>
 
 namespace chip {
 namespace rpc {
 
+::pw::Status TryWriteViaAccessor(const chip::app::ConcreteDataAttributePath & path, chip::app::AttributeValueDecoder & decoder)
+{
+    std::set<AttributeAccessor *> accessors = AttributeAccessorRegistry::Instance().GetAllAccessors();
```

Comment:
Suggest making an interator begin()/end() so that we can write:

```
for (PigweedDebugAccessInterceptor * interceptor : interceptors)
{
    ::pw::Status result = interceptor->Write(path, decoder);
    ...
```